client/src/shared/layouts/CourseSettingsSideBar/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Link from 'next/link';
+
+const CourseSettingsSidebar: React.FunctionComponent = () => {
+  return (
+    <div className="space-y-9">
+      <div>
+        <div>
+          <h1 className="text-lg pb-2 font-medium">Basic Course Settings</h1>
+        </div>
+        <div className="space-y-1 ml-5">
+          <h3 className="underline">
+            <Link href="/">Course Name</Link>
```

Comment:
Please update this accordingly, clicking any anchor in sidebar must jump to its corresponding ID in course-settings

```
<a href="#exampleID">Example</a>

<div id="exampleID">
  ...
</div>
```